src/getMetroBundleOptions.ts

```diff
@@ -0,0 +1,68 @@
+// Copy of https://github.com/react-native-community/cli/blob/13.x/packages/cli-hermes/src/profileHermes/metroBundleOptions.ts as `cli-hermes` was recently removed from React Native Community CLI
+
+import fs from 'fs';
+import type { HermesCPUProfile } from 'hermes-profile-transformer/dist/types/HermesProfile';
+
+export interface MetroBundleOptions {
+  platform: string;
+  dev: boolean;
+  minify: boolean;
+  host: string;
+}
+
+export function getMetroBundleOptions(
+  downloadedProfileFilePath: string,
+  host: string
+): MetroBundleOptions {
+  let options: MetroBundleOptions = {
+    platform: 'android',
+    dev: true,
+    minify: false,
+    host,
```

Comment:
In the original `getMetroBundleOptions` there was no host parameter or return value, and I would remove it from here as well (because actually we are just passing it in and out, and not doing anything with it in the function, so it doesn't have to be a parameter actually).

instead in `getSourcemapFromServer` I would do it as its done in rn-cli, getting the actual ip address:

https://github.com/react-native-community/cli/blob/d4039699f807e9bc3482a3b033ce023e6a5e0d7d/packages/cli-hermes/src/profileHermes/sourcemapUtils.ts#L37-L39

Reply:
We added `host` parameter because `ip` package wasn't maintained and it contained a security vulnerability and library maintainers didn't update it for few weeks 😅 See: https://github.com/react-native-community/cli/pull/2299. IIRC after our testing `localhost` should be safe!

Reply:
okay - are there any use cases we could think of where "localhost" would be incorrect / wouldn't work?

Reply:
I'm not aware of any